README.md

```diff
@@ -19,111 +19,182 @@
+The Quick Start Guide showcases setting up a federation using the TaskRunner API on a single machine (simulation mode)
+|   |  | 
+| -------------- | -------- |
+| Containing 3 Participants | 1. Aggregator  <br/> 2. Collaborator1  <br/> 3. Collaborator2  |
 
-The grant for FeTS was awarded from the [Informatics Technology for Cancer Research (ITCR)](https://itcr.cancer.gov/) program of the National Cancer Institute (NCI) of the National Institutes of Health (NIH), to Dr Spyridon Bakas (Principal Investigator) when he was affiliated with the [Center for Biomedical Image Computing and Analytics (CBICA)](https://www.cbica.upenn.edu/) at UPenn and now heading up the [Division of Computational Pathology at Indiana University (IU)](https://medicine.iu.edu/pathology/research/computational-pathology).
-
-FeTS is a real-world medical federated learning platform with international collaborators. The original OpenFederatedLearning project and OpenFL are designed to serve as the backend for the FeTS platform, and OpenFL developers and researchers continue to work very closely with IU on the FeTS project. An example is the [FeTS-AI/Front-End](https://github.com/FETS-AI/Front-End), which integrates the group’s medical AI expertise with OpenFL framework to create a federated learning solution for medical imaging. 
+**Guide in few simple steps.** _(Tip: use fx --help for any command usage details)_
+```
+0. Setup Prerequisites: Install OpenFL in a conda environment, setup proxies & FQDN.
+1. Setup Federation Workspace & Certificate Authority(CA): Create federation workspace with FL Plan & Certify it as CA.
+2. Setup Aggregator: Generate Aggregator TLS Certificate signed by CA.
+3. Setup Collaborator1: Create & certify collaborator1 workspace.
+4. Setup Collaborator2: Create & certify collaborator2 workspace.
+5. Run the Federation: Start the Aggregator, Collaborator1 & Collaborator2.
+```
+### Step 0: Prerequisites
+Please be sure you have Miniconda or Anaconda installed.
+#### _Setup Miniconda environment_
+```bash
+conda create -n fedai python=3.10 # officially supports versions (>=3.7, <=3.10) 
+conda activate fedai
+```
+#### _Install OpenFL from source_
+```bash
+git clone https://github.com/intel/openfl.git
+python -m pip install -U pip setuptools wheel
+cd openfl/
+python -m pip install .
+python -c "import openfl; print('Successfully installed OpenFL v{}'.format(openfl.__version__))"
+```
+#### _Setup fully qualified domain name [(FQDN)](https://en.wikipedia.org/wiki/Fully_qualified_domain_name)_
+```bash
+echo $(hostname --all-fqdns |  awk '{print tolower($1)}') # Find  the right FQDN
+```
+```bash
+export FQDN=<FQDN> # Enter your FQDN here
+export no_proxy= localhost,<local machine IP>,<FQDN>
+```
+### Step 1: Setup Federation Workspace & Certificate Authority(CA)
+#### _Step 1a: Create a Federation workspace with necessary dependencies & initialize  [FL Plan](https://openfl.readthedocs.io/en/latest/running_the_federation.html#plan-settings.)_
```

Comment:
new link: https://openfl.readthedocs.io/en/latest/about/features_index/taskrunner.html#federated-learning-plan-fl-plan-settings

Reply:
Thank you @kta-intel and for making it easy by sharing the new link.Updated!